showcase/src/Utils.elm

```diff
@@ -40,3 +41,8 @@ iconsSvgSprite =
             []
             []
         )
+
+
+prettifyElmCode : String -> String
```

Comment:
I'm not sure this was the best call, as the notes aren't a simple string but Markdown :smile:

Reply:
the plan here is to use only for `PluginOption.code`